charts/fluent-operator/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: fluent-operator
+description: A Helm chart for Kubernetes
```

Comment:
Can we tidy up the description a bit, this seems a little generic.

We should also add in some of the other metadata fields similar to the FB one: https://github.com/fluent/helm-charts/blob/main/charts/fluent-bit/Chart.yaml
Useful to have links to things.

Reply:
fixed